packages/ui/src/styles/_dnd.scss

```diff
@@ -1,3 +1,5 @@
+@forward '~@patternfly/patternfly/components/DragDrop/drag-drop.scss';
```

Comment:
@mmelko @igarashitm 
We're reexporting the drag-drop.scss styles and variables from patternfly so they are available to the users of this mixin.